### PR TITLE
Issue 32690 : SSHManager with csh shell

### PR DESCRIPTION
**Pull request by DcfGH**
Dear Julia team,

A pull request from issue 32690
Best regards

### CODE REVIEW

**Comment by kshyatt** [comment]
@amitmurthy do you want to review this?

**Comment by ViralBShah** [comment]
@tanmaykm Can you review and merge this? I don't fully understand the intricacies of `csh`, but it looks like a small change.

File: stdlib/Distributed/src/managers.jl [hunk 1]
```diff
@@ -184,10 +184,7 @@ function launch_on_machine(manager::SSHManager, machine, cnt, params, launched,
     tval = get(ENV, "JULIA_WORKER_TIMEOUT", "")
 
     # Julia process with passed in command line flag arguments
-    cmds = """
-        cd -- $(shell_escape_posixly(dir))
-        $(isempty(tval) ? "" : "export JULIA_WORKER_TIMEOUT=$(shell_escape_posixly(tval))")
-        $(shell_escape_posixly(exename)) $(shell_escape_posixly(exeflags))"""
+    cmds = """cd -- $(shell_escape_posixly(dir)) ; $(isempty(tval) ? "" : "export JULIA_WORKER_TIMEOUT=$(shell_escape_posixly(tval))") $(shell_escape_posixly(exename)) $(shell_escape_posixly(exeflags))"""
```
Comment (tanmaykm) [inline stdlib/Distributed/src/managers.jl:187]: If multiple lines were merged, shouldn't there be a command separator after all the lines? I don't see one after the second line.

**Review by tanmaykm — CHANGES_REQUESTED** [review]

**Comment by tanmaykm** [comment]
Since this is quite old, it will be good to rebase this once with master.

We probably do not run our CI tests with `csh`. So we may be only testing for regressions here and not the actual fix. Is there any way to do that?

**Comment by FelipeLema** [comment]
fixes https://github.com/JuliaLang/julia/issues/32690

**Comment by FelipeLema** [comment]
@tanmaykm you can do something like `sudo usermod --shell=/usr/bin/csh travis` (asuming I got the username right).

However, that will make _all_ shells spawned with `csh` instead of default `bash`, which is likely (?) to have consequences midway through the test run.

Is there a proper place where we can set-and-unset `csh` as shell? I was thinking "before remote process tests", is this possible?

**Pull request closed without merging**